README.md

```diff
@@ -1,3 +1,4 @@
 # Google Student Training in Engineering Program
 
 This repo contains Anthony Zhu's portfolio and STEP projects.
+
```

Comment:
Did you mean to add an empty line?

Reply:
Yes, the first commit I made I accidentally did it without making a pull request so I wanted to practice making a PR officially

Reply:
Ack.